testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java

```diff
@@ -69,6 +72,11 @@ public class OrganizationInvitationLinkTest extends AbstractOrganizationTest {
     @Page
     protected RegisterPage registerPage;
 
+    @Before
+    public void setDriverTimeout() {
+        driver.manage().timeouts().pageLoadTimeout(Duration.ofMinutes(1));
```

Comment:
Is it because of the time to send the email ? Or can we remove it?

Reply:
It was originally there: https://github.com/keycloak/keycloak/pull/33334/files#diff-a115c6bafd74ff9e814929582c05e1d3257867062b810c155946449bcb70bf79L227 and I found out that it timeouted few times when I was testing it locally, that's why I increased it to 60s and didn't revert it back. Would you prefer to revert it back?

Reply:
Works for me. Tks.